pkg/cloudprovider/aws/suite_test.go

```diff
@@ -369,6 +354,21 @@ var _ = Describe("Allocation", func() {
 			})
 		})
 		Context("LaunchTemplates", func() {
+			It("should hydrate the launch template cache on instantiation of the LaunchTemplateProvider", func() {
```

Comment:
these honestly aren't very good tests... let me know if you want me to keep these or not. They're in a separate commit, so I can remove all the test mods I made :)